Mathlib/Algebra/Group/Hom/Defs.lean

```diff
@@ -1355,6 +1355,8 @@ theorem coe_one : ((1 : AddMonoid.End A) : A → A) = id := rfl
 theorem coe_mul (f g) : ((f * g : AddMonoid.End A) : A → A) = f ∘ g := rfl
 #align add_monoid.coe_mul AddMonoid.coe_mul
 
+lemma mul_apply (f g : AddMonoid.End A) (a : A) : (f * g) a = f (g a) := rfl
```

Comment:
This lemma has `A` as an explicit argument. That's probably not what is intended.
```suggestion
lemma mul_apply (f g : AddMonoid.End A) (a : A) : (f * g) a = f (g a) := rfl
```

Reply:
I assume this would be:
```
lemma mul_apply {A} [AddZeroClass A] (f g : AddMonoid.End A) (a : A) : (f * g) a = f (g a) := rfl
```
However, with the latest golfing, this lemma is no longer required for this PR, so I've removed it.